src/Module/Util/VaInfo.php

```diff
@@ -401,7 +402,7 @@ public static function get_tag_type($results, $configKey = 'metadata_order')
             'getid3_tag_order' => static::getConfigContainer()->get(ConfigurationKeyEnum::GETID3_TAG_ORDER)
         ];
 
-        $order = (array) ($tagorderMap[$configKey] ?? '');
+        $order = array_map('strtolower', $tagorderMap[$configKey] ?? '');
```

Comment:
for array_map should the empty string be an empty array instead?

Reply:
interesting finding even for me who produced it
$configKey can only be one of the three : metadata_order, metadata_order_video, getid_tag_order
these are hardcoded so if nobody produces a typo there is no chance $tagorderMap[$configKey] is undefined
$order = array_map('strtolower', $tagorderMap[$configKey]); should be sufficient
the other way 
$order = array_map('strtolower', $tagorderMap[$configKey] ?? []);
would presume that something might go wrong and than additional checks would be advidsed

Reply:
I'd say, `array_map('strtolower', $tagorderMap[$configKey] ?? [])` should be sufficient.